benchmark/v8/serialize.js

```diff
@@ -0,0 +1,16 @@
+'use strict';
+
+const common = require('../common.js');
+const v8 = require('v8');
+
+const bench = common.createBenchmark(main, {
+  n: [1e6]
+});
+
+function main({ n }) {
+  const typedArray = new BigUint64Array();
```

Comment:
Is this supposed to be an empty array?

Reply:
This was the recommended change from @mscdex. Do you have any suggestions?

Reply:
Yeah, I would suggest several variations with different sized arrays.

Reply:
Thanks for the review. I added different-sized arrays, and updated the pull request.